core/modules/ccontrol/UserQueryFactory.h

```diff
@@ -32,16 +32,18 @@
   */
 
 // System headers
+#include <cstdint>
 #include <memory>
 
 // Third-party headers
 #include "boost/utility.hpp"
 
-// LSST headers
 #include "lsst/log/Log.h"
+// LSST headers
```

Comment:
Hmm, I wonder who've done that :innocent:  I blame my cat stepping on keyboard :octocat: